hack/install-kwok.sh

```diff
@@ -0,0 +1,102 @@
+set -euo pipefail
```

Comment:
Is this script based off of anything? Or did we write it ourselves? If it's based on some existing script, we should consider linking that script here.

Reply:
I wrote it myself